src/prefect/client/client.py

```diff
@@ -253,6 +261,43 @@ def graphql(
         """
         Convenience function for running queries against the Prefect GraphQL API
```

Comment:
Perhaps with noting that we may initialize the tenant with the first request here